Packages/eu.netherlands3d.twin-features/Scripts/FeatureLink.cs

```diff
@@ -9,7 +10,6 @@ public class FeatureLink
         [HideInInspector]
         public string name;
         public Feature feature;
-        public MonoBehaviour component;
-        public FeatureLinkAction action;
+        public UnityEvent<bool> onFeatureToggle;
```

Comment:
Als je hier een `new()` initializer achter zet; dan hoef je geen nullable checks toe te voegen in invokers. Maakt het geheel net wat type-stricter; en null-checks kosten een (heel) klein beetje performance